tests/target-browser/target_browser_test.go

```diff
@@ -266,405 +250,664 @@ var _ = Describe("Target Browser Tests", func() {
+					cmd.BackupPlanUIDFlag, cmd.BackupUIDFlag)))
+			})
 
-		It(fmt.Sprintf("Should fail if flag %s not given", cmd.BackupUIDFlag), func() {
-			args := []string{cmdGet, cmdMetadata}
-			args = append(args, commonArgs...)
-			command := exec.Command(targetBrowserBinaryFilePath, args...)
-			output, err := command.CombinedOutput()
-			Expect(err).Should(HaveOccurred())
-			Expect(string(output)).Should(ContainSubstring(fmt.Sprintf("required flag(s) \"%s\", \"%s\" not set",
-				cmd.BackupPlanUIDFlag, cmd.BackupUIDFlag)))
+			It(fmt.Sprintf("Should fail if flag %s not given", cmd.BackupUIDFlag), func() {
+				args := []string{cmdGet, cmdMetadata, flagBackupPlanUIDFlag, backupPlanUIDs[0]}
+				args = append(args, commonArgs...)
+				command := exec.Command(targetBrowserBinaryFilePath, args...)
+				output, err := command.CombinedOutput()
+				Expect(err).Should(HaveOccurred())
+				Expect(string(output)).Should(ContainSubstring(fmt.Sprintf("required flag(s) \"%s\" not set",
+					cmd.BackupUIDFlag)))
+			})
+
+			It(fmt.Sprintf("Should fail if flag %s not given", cmd.BackupPlanUIDFlag), func() {
+				args := []string{cmdGet, cmdMetadata, flagBackupUIDFlag, backupUID}
+				args = append(args, commonArgs...)
+				command := exec.Command(targetBrowserBinaryFilePath, args...)
+				output, err := command.CombinedOutput()
+				Expect(err).Should(HaveOccurred())
+				Expect(string(output)).Should(ContainSubstring(fmt.Sprintf("required flag(s) \"%s\" not set",
+					cmd.BackupPlanUIDFlag)))
+			})
+
+			It(fmt.Sprintf("Should fail if flag %s is given without value", cmd.BackupPlanUIDFlag), func() {
+				args := []string{cmdGet, cmdMetadata}
+				args = append(args, commonArgs...)
+				args = append(args, flagBackupPlanUIDFlag)
+				command := exec.Command(targetBrowserBinaryFilePath, args...)
+				output, err := command.CombinedOutput()
+				Expect(err).Should(HaveOccurred())
+				Expect(string(output)).Should(ContainSubstring(fmt.Sprintf("flag needs an argument: %s", flagBackupPlanUIDFlag)))
+			})
+
+			It(fmt.Sprintf("Should fail if flag %s is given without value", cmd.BackupUIDFlag), func() {
+				args := []string{cmdGet, cmdMetadata}
+				args = append(args, commonArgs...)
+				args = append(args, flagBackupUIDFlag)
+				command := exec.Command(targetBrowserBinaryFilePath, args...)
+				output, err := command.CombinedOutput()
+				Expect(err).Should(HaveOccurred())
+				Expect(string(output)).Should(ContainSubstring(fmt.Sprintf("flag needs an argument: %s", flagBackupUIDFlag)))
+			})
+
+			It(fmt.Sprintf("Should get empty  metadata if flag %s is given zero value", cmd.BackupUIDFlag), func() {
+				args := []string{cmdGet, cmdMetadata, flagBackupPlanUIDFlag, backupPlanUIDs[0], flagBackupUIDFlag, ""}
+				args = append(args, commonArgs...)
+				command := exec.Command(targetBrowserBinaryFilePath, args...)
+				output, err := command.CombinedOutput()
```

Comment:
Dont' we get error from server if passed empty values or invalid values?

Reply:
Not getting error from server